Sources/FirebaseAuth/FIRAuthTokenResult.swift

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
+@_exported
```

Comment:
Dumb question perhaps, but I've never used `@_exported` but this directive just means that his library will also export the symbols of the `@_exported` library, is that a correct understanding?

Reply:
No, it simply says that this module is re-exported, which amounts to an `import` in the client of the module.  This is an internal/unsafe attribute, but such it hasn't been made stable yet.